specifications/credential-schema-2022.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Verifiable Credentials JSON Schema 2022",
+  "summary": "Enables JSON Schemas to be applied as a validation method to any portions of Verifiable Credentials",
+  "specification": "https://w3c-ccg.github.io/vc-json-schemas/",
```

Comment:
This would change if the work item is adopted

Reply:
agree, happy to merge now